article-analysis-project/article-processing.py

```diff
@@ -0,0 +1,207 @@
+from __future__ import print_function
+from newspaper import Article
+from genderize import Genderize
+import nltk
+from nltk.tokenize import sent_tokenize
+import gender_guesser.detector as gender
+import spacy 
+import numpy as np
+import json
+import os
+import itertools
+import string
+import urllib
+import requests, mediacloud.api
+from collect_articles import article_urls
+from bs4 import BeautifulSoup
+import re
+import pandas as pd
+def main():
+    urls = article_urls(25499)
+    
+    complete = []
+    for URL in urls:
+        #URL = 'https://abcnews.go.com/Politics/white-house-odd-couple-trump-boltons-tumultuous-relationship/story?id=71323127'
+        article = Article(URL)
+        article.download()
+        article.parse()
+        main_body = article.text #Main body of Article
+        attempted_authors = article.authors
+        sentences = sent_tokenize(main_body)
+
+        #Create genderize object
+        genderize = Genderize(
+            user_agent='Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36',
```

Comment:
It might be good to put this in a `constant` file that gets imported (or configured).

Rule of thumb is that when I see special strings buried in code, it should be pulled and put in a constant or config file.